File: profile/angular/src/app/shared/dto/model/profile-user.ts
import { Address } from './address';
/**
* @class ProfileUser
* 
*/

export class ProfileUser {
    id: number = 0;
    firstName: string = "";
    lastName: string = "";
    email: string = "";
    
    createdDate: Date = new Date();
    lastLoginDate: Date = new Date();
    mailingAddress: Address = new Address();
    physicalAddress: Address = new Address();
    themeId: number = 0;
}

File: profile/maven2/demo/src/main/java/com/example/demo/dto/ProfileUser.java
package com.example.demo.dto;

import com.example.demo.db.model.*;
import java.util.*;
import java.time.LocalDateTime;

/**
* @class ProfileUser
*
*/
public class ProfileUser {
    public long id;
    public String firstName;
    public String lastName;
    public String email;
    
    public LocalDateTime createdDate;
    public LocalDateTime lastLoginDate;
    public Address mailingAddress;
    public Address physicalAddress;
    public long themeId;
}

